### PR TITLE
Hello updates message.

### PR DESCRIPTION
**Pull request by ccobine**
I am opening a pull request. Usually created when the task is complete and are requesting review. Also can create when you need assistance. link to JIRA if relevant.

### CODE REVIEW

File: index.txt [hunk 1]
```diff
@@ -1 +1,5 @@
-hello-world
\ No newline at end of file
+hello-world
```
Comment (heyawhite) [inline index.txt]: Why is there a hyphen?

Comment (ccobine) [reply index.txt]: Because I like it?

Comment (heyawhite) [reply index.txt]: FINE BE THAT WAY.

**Comment by heyawhite** [comment]
Looks good!

**Pull request merged**